src/main/resources/hudson/model/ExternalJob/newJobDetail.jelly

```diff
@@ -24,5 +24,5 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <div>
-  ${%body}
+  ${it.description}
```

Comment:
It was addressed here https://github.com/jenkinsci/external-monitor-job-plugin/pull/9/commits/56957b390b384b73dcf9abd7fba1199b0b6d71f3